locales/result/br.json

```diff
@@ -87,7 +87,7 @@
   "mail.assign.remove.message": "<p>Você foi removido da tarefa <a href=\"{{url}}\">{{title}}</a> no Gitpay</p><p>O responsável pela tarefa enviou a seguinte mensagem:</p><p>{{message}}</p>",
   "mail.assign.remove.owner.subject": "O usuário foi desassociado de uma tarefa que você criou no Gitpay",
   "mail.assign.remove.owner.message": "<p>O usuário {{user}} ({{email}}) foi desassociado da tarefa <a href=\"{{url}}\">{{title}}</a> no Gitpay</p><p>A razão foi:</p><p>{{message}}</p>",
-  "mail.invite.send.action": "Você recebeu um convite para trabalhar em uma tarefa do Gitpay :-)",
+  "mail.invite.send.action": "Você recebeu um convite de {{name}} para trabalhar em uma tarefa do Gitpay :-)",
```

Comment:
@spikeboom please do not change here cause this is a generated file with the final translations.

Reply:
solved